pkgs/servers/microserver/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+   pname = "microserver";
+   version = "0.1.8";
+
+  src = fetchFromGitHub {
+    owner = "robertohuertasm";
+    repo = "microserver";
+    rev = "v${version}";
+    sha256 = "1i9689ra5jnmhkxabrx4zcp5f422w9ql9m4xzldqwmpnckm736v6";
+  };
+
+  cargoSha256 = "0ldn7g154ncg0h10442p1kr4nmx63wbws74ywm6z64g8kvciv54m";
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/robertohuertasm/microserver;
```

Homepage should be quoted per https://github.com/NixOS/rfcs/pull/45.
```suggestion
    homepage = "https://github.com/robertohuertasm/microserver";
```